cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('should just work', () => {
```

Comment:
if we're going to get this merged, perhaps we could reword the description to be `should have a visible play button with video duration`

Reply:
Well spotted. My bad!